docs/changelog/99584.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 99584
+summary: Platform specific models
```

Comment:
```suggestion
summary: Adding an option for trained models to be platform specific
```

Reply:
Do I also need to change the title of this PR, or can I just change the yaml file directly?

Reply:
Just this file is enough. This file defines what goes in the release notes. The PR title is just what the bot uses as an initial idea.